media/js/base/mozilla-client.js

```diff
@@ -321,72 +322,113 @@ if (typeof Mozilla === 'undefined') {
     };
 
     /**
-     * Use the async mozUITour API of Firefox to retrieve the user's Sync/FxA info. This API
-     * is available on Firefox 29 and later. See
+     * Use the async mozUITour API of Firefox to retrieve the user's FxA info. See
      * http://bedrock.readthedocs.org/en/latest/uitour.html for details.
      *
-     * @param  {Function} callback - callback function to be executed with the Sync details
+     * FxA was previously branded as Sync.
+     *
+     * The various states here are... complicated
+     * This is the intention:
+     *     - firefox: true if Firefox
+     *     - legacy: true if older than FxALastSupported
+     *     - mobile: false | android | ios
```

Comment:
I updated the mobile variable to indicate if it is android or ios and treat both as "Firefox Mobile" now.